src/core/sys/windows/commctrl.d

```diff
@@ -3869,7 +3869,7 @@ static if (_WIN32_WINNT >= 0x501) {
     }
     alias LVINSERTGROUPSORTED* PLVINSERTGROUPSORTED;
 
-    alias int function(INT, INT, VOID*) PFNLVGROUPCOMPARE;
+    extern (Windows) alias int function(INT, INT, VOID*) PFNLVGROUPCOMPARE;
```

Comment:
I think the change of styles should be made at all.
So, these core.sys.windows.* modules have thousands of aliases, mostly in old styles.
I misses such a skill to do efficiently :P